pkg/kubelet/pod_workers.go

```diff
@@ -28,7 +28,15 @@ import (
 	"github.com/golang/glog"
 )
 
-type syncPodFnType func(*api.Pod, *api.Pod, kubecontainer.Pod) error
+const (
+	minBackoffInterval = time.Second * 4
```

Comment:
I think it might be a bit misleading to call it sync interval because the actual sync is triggered by updates and periodic sync intervals defined elsewhere. I'm open to other names though :)